certified-connectors/TPS & CTPS Checker/apiDefinition.swagger.json

```diff
@@ -0,0 +1,168 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "API for Querying the TPS/CTPS status of a UK telephone number",
+    "title": "TPS & CTPS Checker",
+    "version": "1.0",
+    "contact": {
+      "name": "QuartzData",
+      "url": "https://tpsmydata.co.uk/#contact",
+      "email": "info@quartzdata.co.uk"
+    }
+  },
+  "host": "api.tpsmydata.co.uk",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/",
+  "paths": {
+    "/": {
+      "get": {
+        "summary": "Query the TPS CTPS status of UK based telephone number",
```

Comment:
Kindly add description property as per swagger validation error below. 

The 'description' property is required. ValuePath: /paths/~1/get

Please refer the swagger validation schema here - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/apiDefinition.swagger.schema.json